modules/dnn/src/onnx/onnx_graph_simplifier.cpp

```diff
@@ -767,6 +767,25 @@ Mat getMatFromTensor(const opencv_onnx::TensorProto& tensor_proto)
             Mat(sizes, CV_32FC1, val).copyTo(blob);
         }
     }
+    else if (datatype == opencv_onnx::TensorProto_DataType_FLOAT16)
+    {
+        // FIXME, for now, we only load FP16 Tensor as FP32 Mat, full support for FP16 is required in the future.
+        static bool warningOnce = true;
```

Comment:
If the end user decides to import two networks, it'll see only one warning. I think it's okay to spam this on every converted tensor during import.

Reply:
Hi @rogday. A model may output hundreds of lines of warning, which is too messy. I think one line of warning is enough to let the user know what's going on inside.

Reply:
Then it should be one line per model import imo.